src/main/java/com/ecommerce/ECommerceApp/model/Seller.java

```diff
@@ -8,27 +9,27 @@
 
 public class Seller{
 
-    private List<int> productIDList;
-    private List<int> orderFulfillment;
+    private List<Integer> productIDList = new ArrayList<>();
+    private List<Integer> orderFulfillment = new ArrayList<>();
```

Comment:
I converted the primitive type of int to Integer (a wrapper class) so it adheres to java standards, as List utilizes reference types and not primitive types. 

Source: https://www.tutorialspoint.com/java/java_documentation.htm